src/github-utils.ts

```diff
@@ -37,3 +37,28 @@ export async function getChecks(): Promise<CheckRun[]> {
 	}
 	return response.check_runs;
 }
+
+// https://docs.github.com/en/rest/reference/git#create-a-reference
+export async function createRef(tag: string, sha: string): Promise<string> {
+	const token = core.getInput('github_token', { required: true });
+	const octokit = github.getOctokit(token);
+
+	core.debug(`Creating refs/tags/${tag}`);
+
+	return await octokit.rest.git
+		.createRef({
+			owner: github.context.repo.owner,
+			repo: github.context.repo.repo,
+			ref: `refs/tags/${tag}`,
+			sha,
+		})
+		.then((response) => {
+			core.debug(`Created ${response.data.url}`);
+			return response.data.url;
+		})
+		.catch((e) => {
+			core.error(e.message);
+			// TODO: catch error if not 'ref already exists'
+			return '';
+		});
```

Comment:
Same comments as the `getReleaseVersion` function. However, Here is how I would do both functions so use this example as guide for the release version function.

```suggestion
	const response = await octokit.rest.git
		.createRef({
			owner: github.context.repo.owner,
			repo: github.context.repo.repo,
			ref: `refs/tags/${tag}`,
			sha,
		});
	// TODO: catch error if not 'ref already exists'	
	core.debug(`Created ${response.data.url}`);	
	return response.data.url;	
```

Only catch exception if you will be adding some handling (such as dealing with ref already exists) or want to log a better message.